frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -90,15 +95,17 @@ const tableHeaders = (hasMultiplexResults: boolean) => (
         </th>
       </>
     ) : null}
-    <th scope="col" className="test-facility-cell">
-      Testing facility
-    </th>
     <th scope="col" className="test-device-cell">
       Test device
     </th>
     <th scope="col" className="submitted-by-cell">
       Submitted by
     </th>
+    {displayFacility && (
```

Comment:
nit: for the sake of consistency/style, we might consider using the same tenary syntax used on line 88 in the event of displayFacility being false. Similar feedback for line 236.